satpy/tests/test_utils.py

```diff
@@ -486,14 +486,28 @@ def test_chunk_size_limit():
     from unittest.mock import patch
 
     from satpy.utils import get_chunk_size_limit
-    with patch("satpy.utils.CHUNK_SIZE", None):
-        assert get_chunk_size_limit(np.uint8) is None
     with patch("satpy.utils.CHUNK_SIZE", 10):
         assert get_chunk_size_limit(np.float64) == 800
     with patch("satpy.utils.CHUNK_SIZE", (10, 20)):
         assert get_chunk_size_limit(np.int32) == 800
 
 
+def test_chunk_size_limit_from_dask_config():
+    """Check the chunk size limit computations."""
+    from unittest.mock import patch
+
+    import dask.config
+
+    from satpy.utils import get_chunk_size_limit
+    with patch("satpy.utils.CHUNK_SIZE", None):
+        original_chunk_size = dask.config.get('array.chunk-size')
+        try:
+            dask.config.set({"array.chunk-size": "1KiB"})
```

Comment:
I may be misunderstanding this, but why not `with dask.config.set(...):`?

Reply:
ah, good point, I forgot that was a possibility